AGP-71: Nest 2019 Budget Extension (#71)
aragon/AGPs: AGPs/AGP-XX.md

| AGP | Title | Author | Status | Track   | Created |
|-----|-------|---------------------------|--------|---------|---------|

> | XX  | Nest budget |   louisgrx |      | Finance | 2019-07-10       |

Reviewer: Is the AGP title "Nest 2019 Budget Extension" or "Nest budget"?

Author: Nest 2019 Budget Extension, will correct

# AGP-XX: Nest 2019 Budget Extension

## Address of the transfer recipient

Nest DAO: https://mainnet.aragon.org/#/nest/0x4cbabbe13cc4b0c724e7dcecd6ae56ae1905efb2

## Amount of the transfer

> $600k in DAI 

Reviewer: ```suggestion
600,000 DAI 
```

> ANT300k

Reviewer: ```suggestion
300,000 ANT
```

## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)

> A one-off transfer

Reviewer: ```suggestion
1
```

## Purpose of the transfer

#### Use of funds
The funds received will be managed by the [Nest program](https://github.com/aragon/nest) and allocated to grantees. 

> - $500K DAI  to cover the cost of their operations

Reviewer: ```suggestion
- 500,000 DAI  to cover the cost of their operations
```

> - $100K DAI to cover the cost of audits of successful projects

Reviewer: ```suggestion
- 100,000 DAI to cover the cost of audits of successful projects
```

> - ANT300K to reward grantees and invite them to participate actively in Aragon

Reviewer: ```suggestion
- 300,000 ANT to reward grantees and invite them to participate actively in Aragon
```

#### Following up on the [AGP-17](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-17.md) budget

> In the first half of 2019, the Nest program allocated 3 grants. Considering the current pipeline of high quality proposals, it is expected by the Aragon Association that all remaining funds will have dried up by ANV#3.

Reviewer: ```suggestion
In the first half of 2019, the Nest program allocated 3 grants. Considering the current pipeline of high quality proposals, it is expected by the Aragon Association that all remaining funds will have dried up by ANV-3.
```

Reviewer: The shorthand convention the community seems to have converged on is `ANV-[#]` e.g. ANV-3.

#### The state of Nest

In the past year, Nest has found a great use case in funding small and agile teams, working from 6 months to a year, on technologies or initiatives that hold a unique place in the Aragon ecosystem. The Nest 2019 grantees are a good representation of this market-fit for Nest. 
- The Aragon Mesh team is working full time on enhancing the aragonCLI which is a central developer tool
- The LevelK team is on its second Nest grant to build and implement powerful governance tools such as its Futarchy app
- The Frame team is developing a signing provider with integrations with the Aragon stack and aims at improving the ease of use of Aragon DAOs

> Moreover, the Nest program has proven to be a great way for Aragon community members to ramp up their contribution to Aragon. Two past Nest teams are now Flock teams, and it is expected that Nest will keep on being a great way to attract talent to contribute on the long term. It is also clear that we're starting to see an interesting ecosystem dynamic with Nest as a bridge between Flock and the broader community.

Reviewer: ```suggestion
Moreover, the Nest program has proven to be a great way for Aragon community members to ramp up their contribution to Aragon. Two past Nest teams are now Flock teams, and it is expected that Nest will continue being a great way to attract talent to contribute for the long term. It is also clear that we're starting to see an interesting ecosystem dynamic with Nest as a bridge between Flock and the broader community.
```

> Finally, while many things remain to be done to make Nest the most appealing of all funding programs, the Aragon Association will keep doing its best to provide an ever better experience to Nest grantees in order to nurture Aragon's vision in them. As a first step in this direction, the Aragon Association will propose the decentralization of Nest with the Nest DAO through an AGP in ANV#3.

Reviewer: ```suggestion
Finally, while many things remain to be done to make Nest the most appealing of all funding programs, the Aragon Association will continue doing its best to provide an ever better experience to Nest grantees in order to nurture Aragon's vision in them. As a first step in this direction, the Aragon Association will propose the decentralization of Nest with the Nest DAO through an AGP in ANV-3.
```

> **The new funding requested in this AGP is necessary for Nest to run efficiently, at least until ANV#4.**

Reviewer: ```suggestion
**The new funding requested in this AGP is necessary for Nest to run efficiently, at least until ANV-4.**
```

#### The Nest DAO

> Provided AGP-XX passes. The Nest DAO will have full control over the grant amounts (in DAI/ANTs) and the selection of grantees. Otherwise, the Aragon Association will keep running the program.

Reviewer: ```suggestion
Provided AGP-XX passes, the Nest DAO will have full control over the grant amounts (in DAI/ANT) and the selection of grantees. Otherwise, the Aragon Association will continue running the program.
```
